client/clientservice/include/client/clientservice/client_service.hpp

```diff
@@ -28,21 +28,30 @@ class ClientService {
       : logger_(logging::getLogger("concord.client.clientservice")),
         client_(std::move(client)),
         event_service_(std::make_unique<EventServiceImpl>(client_)),
-        request_service_(std::make_unique<RequestServiceImpl>(client_)),
         state_snapshot_service_(std::make_unique<StateSnapshotServiceImpl>(client_)){};
 
-  void start(const std::string& addr, uint64_t max_receive_msg_size);
+  void start(const std::string& addr, int num_async_threads, uint64_t max_receive_msg_size);
```

Comment:
Nit: I think `num_async_threads` can be an unsigned type.

Reply:
Thanks, will fix in a follow-up.